backend/typescript/services/implementations/camperService.ts

```diff
@@ -637,6 +640,55 @@ class CamperService implements ICamperService {
     }
   }
 
+  /* eslint-disable consistent-return */
+  async updateWaitlistedCamperStatus(waitlistedCamperId: string): Promise<any> {
```

Comment:
nit: can we update the name of this function to `inviteWaitlistedCamper`? a bit more descriptive than updateWaitlistedCamperStatus imo